### ISSUE TITLE
No error dialog appears when launching Storage Explorer without Linux dependencies installed

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#2702]
**Storage Explorer Version:** 1.12.0
**Build**: [20200309.6](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3537547)
**Branch**: master
**Platform/OS:** Linux Ubuntu 16.04
**Architecture**: x64
**Regression From:** Previous release (1.12.0)

**Steps to reproduce:**
1. Create a  Linux virtue machine with OS (Ubuntu 16.04).
2. Download build -> Select *tar folder -> Extract.
3. Open terminal -> Navigate to where Storage Explorer is located.
4. Run ./StorageExplorer to open Storage Explorer.
5. Check the result.

**Expect Experience:**
An error dialog appears once the Storage Explorer is opened.
![image](https://user-images.githubusercontent.com/41351993/76299853-ba949d00-62f6-11ea-9abb-fabdb2824a53.png)

**Actual Experience:**
1. No error dialog appears.
2. The resources on the left side always keep loading.

**Comment by craxal** [comment]
@v-xianya Does this also reproduce on Ubuntu 18.04?

Been attempting to investigating myself, but I'm blocked by #1887.

**Comment by v-xianya** [comment]
@craxal 

Verified on Ubuntu 18.04 & 19.04. The below error occurs when trying to launch Storage Explorer without any dependencies.
![image](https://user-images.githubusercontent.com/41351993/76817945-4058a100-683f-11ea-9d91-b6b6f990a098.png)

**Comment by craxal** [comment]
@v-xianya That's a different error message than the .NET one, which suggests this bug may be a duplicate of #1887. For that bug, we're trying to figure out which libraries need to be installed for the password message to go away. Can you go back and do the following for each Linux version:
1. Install .NET Core first.
2. Install the additional dependencies one by one until error messages go away and resources load.
3. List the dependencies you installed to get things working.

**Comment by v-xianya** [comment]
@craxal 
Hi Craig, I sent an email to you about the Linux dependencies.

**Comment by craxal** [comment]
@v-xianya It didn't sound like this bug was reproducible after running tests to verify Linux dependencies. Can you verify if this issue still reproduces just to be sure?

**Comment by v-xianya** [comment]
Hi @craxal Verified on build master/20200610.10. This issue doesn't reproduce.
 
Run Storage Explorer without .NET Core installed. The now behaviors is:
Pop up the below dialog after clicking ‘I Accept’ in the prompt '**End-User License Agreement**’ dialog.
![image](https://user-images.githubusercontent.com/41351993/84454724-fc363a80-ac8d-11ea-88a4-1256afe6ceb5.png)
